pkg/apis/apps/install/install.go

```diff
@@ -37,7 +37,7 @@ func Install(groupFactoryRegistry announced.APIGroupFactoryRegistry, registry *r
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:                  apps.GroupName,
-			VersionPreferenceOrder:     []string{v1beta1.SchemeGroupVersion.Version, v1beta2.SchemeGroupVersion.Version},
+			VersionPreferenceOrder:     []string{v1beta2.SchemeGroupVersion.Version, v1beta1.SchemeGroupVersion.Version},
```

Comment:
we cannot change the storage version to v1beta2 until 1.9 (TestEtcdStoragePath integration test will fail if this affected the storage version)